### DISCUSSION TITLE
where is   xsdk/r5_0_bsp/psu_cortexr5_0/include  ?

### DISCUSSION THREAD

**Discussion by wangyong5123** [OpenAMP/libmetal#192] [Q&A]
where is  xsdk/r5_0_bsp/psu_cortexr5_0/include ?

**Comment by arnopo** [comment]
@edmooring 
Could you help? is it a Xilinx point or a libmetal one?

**Comment by edmooring** [comment]
This is a Xilinx-specific issue. The Xilinx SDK creates a BSP (libraries and include files) specific to the board and the design for that board. Where that BSP gets created is up to the user to some extent. Without more context, I can't give a more complete response.
